fiat-ldap/src/test/groovy/com/netflix/spinnaker/fiat/roles/ldap/LdapUserRolesProviderTest.groovy

```diff
@@ -32,10 +32,6 @@ import javax.naming.directory.SearchControls
 
 class LdapUserRolesProviderTest extends Specification {
 
-  @Shared
```

Comment:
looks like Shared and Subject are unused imports now.

Reply:
Will remove.